internal/services/policy/remediation_management_group.go

```diff
@@ -92,12 +92,9 @@ func resourceArmManagementGroupPolicyRemediation() *pluginsdk.Resource {
 				},
 			},
 
-			"policy_definition_id": {
+			"policy_definition_reference_id": {
```

Comment:
Thanks @gettek, however we cannot just remove the old property/ we'll need to deprecate it and then wrap it in a features.FourPointOhBeta flag.

Reply:
Thanks @katbyte properties added back with deprecation message

Reply:
Thanks @katbyte properties added back with deprecation message